lib/db/mysql/index.js

```diff
@@ -299,7 +300,7 @@ MysqlStore.prototype = {
     if (!client.id) {
       return P.reject(new Error('Update client needs an id'));
     }
-    var secret = client.hashedSecret || client.secret || null;
+    var secret = client.hashedSecret;
```

Comment:
Did you want to rename this to `var hashedSecret = ...` too? (as well as lines 304, 305, and 311)